images/conduit/opensearch/entrypoint.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+## SETUP Connection Profile
+mkdir -p /root/.opensearch-cli/
+
+echo $CONNECTION_SECRET | gawk '{
+  match($0, /^https:\/\/([A-Za-z0-9_]+):([^@]+)@(.+)$/, arr);
+  print "profiles:"
+  print "    - name: connection"
+  print "      endpoint: \"https://"arr[3]"\""
+  print "      user: \""arr[1]"\""
+  print "      password: \""arr[2]"\""
+}' > /root/.opensearch-cli/config.yaml
+
+chmod 600 /root/.opensearch-cli/config.yaml
```

Comment:
Should we have a comment here to explain why we set this to `600`? It won't work if it's something else.

Reply:
Added comment here